### PR TITLE
Fix - missing common storage for Nextcloud and Jellyfin / services in settings.yaml

### PR DESCRIPTION
**Pull request by glevil**

### CODE REVIEW

File: src/build/configure-guest.sh [hunk 1]
```diff
@@ -77,6 +77,12 @@ prepare () {
       state: directory
       mode: 0755
     loop: "{{ ['dhcpd', 'traefik', 'authelia', 'adguardhome', 'netdata', 'heimdall', 'database', 'nextcloud', 'jellyfin'] | product(['config', 'data']) | list }}"
+  - name: ensure media directories exist
+    file:
+      path: "/pinanas/dist/{{ item[0] }}/{{ item[1] }}"
+      state: directory
+      mode: 0760
+    loop: "{{ ['media'] | product(['movies', 'tvshows']) | list }}"
```
Comment (yscialom) [inline src/build/configure-guest.sh:85]: Duplicate with [src](https://github.com/yscialom/pinanas/tree/develop/src)/[tasks](https://github.com/yscialom/pinanas/tree/develop/src/tasks)/[jellyfin](https://github.com/yscialom/pinanas/tree/develop/src/tasks/jellyfin)/main.yaml

File: src/templates/docker-templates/jellyfin.yaml [hunk 2]
```diff
@@ -35,4 +35,5 @@ jellyfin:
     - ./jellyfin/data/misc/metadata:/data/metadata
     - ./jellyfin/data/misc/cache:/data/cache
     - ./jellyfin/data/misc/databases:/config/data/data
+    - ./media:/data/media
```
Comment (yscialom) [inline src/templates/docker-templates/jellyfin.yaml:38]: Should replace `/data/movies` and `/data/tvshows` defined 5 lines up?

**Review by yscialom — CHANGES_REQUESTED** [review]

File: src/templates/docker-templates/jellyfin.yaml (hunk 2 above)
Comment (glevil) [reply src/templates/docker-templates/jellyfin.yaml:38]: Yes.

File: src/build/configure-guest.sh (hunk 1 above)
Comment (glevil) [reply src/build/configure-guest.sh:85]: I will edit this file.

File: test/test-traefik-dashboard.sh [hunk 3]
```diff
@@ -27,6 +27,7 @@ web_expect "https://${traefik}" -c 302 -r "https://${traefik}/dashboard/"
 web_expect "https://${traefik}/dashboard/" -c 200
 
 # api
+sleep 60
```
Comment (yscialom) [inline test/test-traefik-dashboard.sh]: See [`.github/workflows/continuous-integration.yaml:60`](https://github.com/yscialom/pinanas/blob/develop/.github/workflows/continuous-integration.yaml#L60)

File: src/build/configure-guest.sh (hunk 1 above)
Comment (yscialom) [reply src/build/configure-guest.sh:85]: Could be simplified + mode 0770 would be preferred:
```yaml
  - name: ensure media directories exist
    file:
      path: "/pinanas/dist/media/{{ item }}"
      state: directory
      mode: 0770
    loop:
    - movies
    - tvshows
```

Comment (yscialom) [reply src/build/configure-guest.sh:85]: FYI mode 0770 versus 0760 means the group (docker) will gain the eXecution rights. For directories, this right is needed to list (as in `ls`) its content.

Comment (yscialom) [reply src/build/configure-guest.sh:85]: FYI2 mode traditionally is written with a leading zero to denote it is a number in base 8. This leading zero is sometimes used to carry a special right, called "the sticky bit".

To read the mode `0ugo`:
- `u`: user's permission (file owner)
- `g`: group's permission
- `o`: other's permission
where each digit is the sum of the following rights:
- `r`, 4: read
- `w`, 2: write
- `x`, 1: execute.

So `0760` means user can do all (4+2+1), group can read and write (4+2) but cannot list the content, others cannot do anything (0).

**Review by yscialom — CHANGES_REQUESTED** [review]

**Review by yscialom — APPROVED** [review]

**Pull request merged**